alonzo/impl/src/Cardano/Ledger/Alonzo/TxBody.hs

```diff
@@ -98,6 +94,7 @@ newtype IsFee = IsFee Bool
       FromCBOR
     )
 
+{-
```

Comment:
since `TxIn` will be the same in Shelley and Alonzo, can't we delete this commented code?

Reply:
done